ROBT 3341 - SCARA/scara_control.c

```diff
@@ -86,12 +85,20 @@ void homeRobot(void) {
 /*|moveJoint|------------------------------------------------------------------
 #
 # Decription:
-#	This function prompts the user for a Joint 1 and Joint 2 position.
+#	This function prompts the user for a Joint 1 and Joint 2 position, then moves there
 #
-# Last Modified: September 30, 2025
+# Last Modified: Dec 02, 2025 - Peter

```

Comment:
This is a waste of time to track, Git will let you know when every single line in every single file has been updated.  You should just removed any and all datestamps from comments